test/test-cases.txt

```diff
@@ -90,3 +90,9 @@
 # Text following a <br>
 {"blocks":[{"type":"unstyled","entityNodes":[{"entity":null,"styleNodes":[{"text":"Hello\nWorld!","styles":null}]}]}]}
 <p>Hello<br /> World!</p>
+
+# HTML video with iframe
+{"entityMap":{"0":{"type":"VIDEO","data":{"src":"https://www.youtube.com/embed/sFXQwk63Y9E?showinfo=0&rel=0", "width":"400", "height":"300"}}},"blocks":[{"type":"unstyled","entityNodes":[{"entity":"0","styleNodes":[{"text":"</p>","styles":null}]}]}]}
```

Comment:
@sstur I change `<p>` to `<div>` instead but still get `text: "</div>`. Am I missing something here?